PRChecker/Shared/FilterView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  FilterView.swift
+//  PRChecker
+//
+//  Created by Chen Yuhan on 2021/11/06.
+//
+
+import SwiftUI
+
+struct FilterView: View {
+    var body: some View {
+        VStack {
+            Header()
+                .padding(EdgeInsets(top: 16, leading: 0, bottom: 8, trailing: 8))
+            Divider()
```

Comment:
Is this something you made or a premade class I missed? It's definitely useful!

Reply:
`Divider` is included in SwiftUI~!